sim/core/consumes.go

```diff
@@ -122,48 +126,67 @@ func addImbueStats(character *Character, imbue proto.WeaponImbue, isMh bool, sha
 
 		// Sharpening Stones
 		case proto.WeaponImbue_SolidSharpeningStone:
-			weapon := character.AutoAttacks.MH()
-			if !isMh {
-				weapon = character.AutoAttacks.OH()
+			if !character.PseudoStats.FeralCombatEnabled {
```

Comment:
Since all of these seem to not work for feral could we instead just add `if !character.pseudoStats.FeralCombatEnabled` outside of the `case` statement so there's only 1?

Reply:
Oh nvm I see that it's not 🫠